### PR TITLE
[10.0][MIG] account_type_multi_company

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

File: account_type_multi_company/README.rst [hunk 1]
```diff
@@ -1,20 +1,23 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
```
Comment (elicoidal) [inline account_type_multi_company/README.rst]: better use png image following new standards

File: account_type_multi_company/README.rst [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
 
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License: AGPL-3
-
-This module was written to extend the functionality of account types
-and make them multi company aware.
-
-It also adds an active flag on account types.
+This module makes the account types multi company aware.
```
Comment (elicoidal) [inline account_type_multi_company/README.rst]: multi-company

File: account_type_multi_company/README.rst [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
 ============
 
-There is no specific installation procedure for this module.
+The functionality added by this module becomes more easily available
+when installing the 'account_type_menu' module.
+
+Obsolete account types defined by e.g. installing multiple
+localisation modules can be hidden via the 'account_type_inactive' module.
+
+Both modules can downloaded from git, cf. https://github.com/OCA/account-financial-tools/tree/10.0
```
Comment (elicoidal) [inline account_type_multi_company/README.rst]: can be

**Review by elicoidal — CHANGES_REQUESTED** [review]
Some details. Other than that LGTM

**Comment by luc-demeyer** [comment]
@elicoidal 
Thanks for the review.
I have fixed the typos.
Changing the svg to png doesn't seem a good idea since
https://img.shields.io/badge/license-AGPL--3-blue.png
does not exist.

**Comment by elicoidal** [comment]
@luc-demeyer are you sure? on my side there is an output.
screenshot from 2017-11-14 09-05-32

and wget works fine.

Is there something I am missing

FYI: we removed the svg because it is not rendered properly in the Odoo apps platform

**Comment by luc-demeyer** [comment]
@elicoidal 
strange but now the png renders fine effectively.
I have updated the PR.

**Review by elicoidal — APPROVED** [review]

**Review by lasley — APPROVED** [review]

**Comment by lasley** [comment]
Thanks @luc-demeyer

**Pull request merged**